src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom";
 import { BrowserRouter, Route } from "react-router-dom";
 import { Security, LoginCallback } from "@okta/okta-react";
 
+import ExampleList from "./components/ExampleListComponent"
```

Comment:
The rest of these have semi-colons. Maybe we need to tweak the linter/prettier config to either do semi's or not. 

It also sparks the conversation of other opinionated conventions we ought to be using.